### PR TITLE
[10] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by elizabethbusola**
## Task: To create & Style the customer signup page using CSS
- I created a signup page and I styled it with External CSS styling method
mobile
tablet
webpage
webpage2

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @elizabethbusola 

The form on the mobile view looks too compressed

**Comment by adaugochi** [comment]
and contact and signup form should be in the `htmlform` folder

**Comment by adaugochi** [comment]
You have conflict

**Comment by elizabethbusola** [comment]
> You have conflict

I have resolved it

File: submissions/busolaAdedoyin/css/style.css [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+ ul {
+    padding: 0;
+    margin: 0;
+ }
+
+ header {
+    background:#330066;
+    color: white;
+    padding-top: 10px;
+    min-height: 15%;
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/css/style.css:24]: using px for height is better

**Pull request merged**